### ISSUE TITLE
jitsi issues on opera

### ISSUE THREAD

**Issue by rachelbian** [calvinckho/restvo-app#94]
I've been having some issues connecting to our video call platform from opera web browser. There's been one time it's worked (which is the most recent time I've used it) and that was the most recent team sync, so maybe this issue has been fixed. What happens is that I join, and it looks like people are there but I'm unable to see video and it just looks like everyone is waiting. I'll see visual cues that someone is speaking but no one actually is. From other people's perspective, I think it looks like I've lost connection.

Version:68.0.3618.125
System:Mac OS X 10.13.6 64-bit

**Comment by calvinckho** [comment]
thank you Rachel for reporting this bug!

**Comment by calvinckho** [comment]
@Ananya157 can you also check Jitsi on the opera browser?

**Comment by calvinckho** [comment]
@rachelbian There is a new version of Jitsi API. can you check on your opera browser and see if this still happens?

**Comment by calvinckho** [comment]
@kevin-tolentino when you get a chance, can you test our video call feature Jitsi on Opera?

**Comment by kevin-tolentino** [comment]
@calvinckho, yes I'll take a look at it and get back to you

**Comment by calvinckho** [comment]
@kevin-tolentino when you get a chance, please let me know update on this. Thanks

**Comment by kevin-tolentino** [comment]
@calvinckho, I tested it out the video conferencing feature just now using the Opera Browser on my PC (Surface Laptop 2) and my phone (Samsung Note 9). I was logged in as the Ted Ho test account on the Opera Browser, and as myself on my phone app. I tested all of the issues Rachel described, and there doesn't seem to be any problems. I forgot to use the Opera Browser during our team video conf this week. I can do it next Wed. if you'd like.

Feel free to refer to this video recording of me testing out the Opera Browser. There were no issues with a two-person conference: https://www.loom.com/share/69aa7dd4493c48319b88ec5b46bbe426?from_recorder=1

Let me know if you have any questions!
